services/response_code.proto

```diff
@@ -1529,4 +1529,9 @@ enum ResponseCodeEnum {
      * The node account is not allowed to be updated
      */
     UPDATE_NODE_ACCOUNT_NOT_ALLOWED = 359;
+
+    /*
+     * The token has no metadata or supply key
+     */
+    TOKEN_HAS_NO_METADATA_OR_SUPPLY_KEY = 360;
```

Comment:
This will conflict with values in this [HIP-904 PR](https://github.com/hashgraph/hedera-protobufs/pull/389). It might be helpful to coordinate the two changes to avoid unnecessary merge conflicts.

Reply:
Agreed, I think that PR will be merged after this. Will comment on that PR